.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -20,9 +20,6 @@ on:
   push:
     branches:
       - 'renovate/main-**'
-  # Run every 8 hours
-  schedule:
-    - cron:  '0 5/8 * * *'
```

Comment:
Should we drop the `push:` target too? If the test shouldn't run on main then matching renovate main branch prefixes doesn't make sense. When preparing this workflow for stable branches, we would substitute that to be `renovate/v1.18-**` but that doesn't seem quite right either.

Reply:
Why would we only run this workflow when renovate pushes a PR for a stable branch? What's special about renovate PRs that might require this that other PRs wouldn't?

Reply:
> Ooh, is that how that works? I see.

I mean, that's what I'm guessing :)

> So do I conclude then that for renovate PRs against main we don't need to run this but you would like to run it for renovate PRs against stable branches?

Yes.

Reply:
Keeping this as-is would be simplest for the branching process just because there's already a step that would re-enable this workflow for the stable branch renovate PRs. The trade off is this will run the workflow unnecessarily on renovate PRs to main. I think I'm fine with that unless you have an idea how to disable this trigger and set up the branching process to re-enable the workflow when we make the stable branch.